lib/base/tlsutility.cpp

```diff
@@ -249,6 +249,49 @@ int ResolveTlsProtocolVersion(const std::string& version) {
 	}
 }
 
+Shared<boost::asio::ssl::context>::Ptr SetupSslContext(String certPath, String keyPath,
+	String caPath, String crlPath, String cipherList, String protocolmin, DebugInfo di)
```

Comment:
There's already another function using the same name in this file: https://github.com/Icinga/icinga2/blob/9e5be6ad9db319e9c51fb7e5933e4b58336c66bc/lib/base/tlsutility.cpp#L73